tests/ernm_terms_test.R

```diff
@@ -0,0 +1,459 @@
+# This script carries out testing for the newly coded up ERNM terms:
```

Comment:
I'm not sure that I set up tests correctly here. Are the cpp tests even being run?  We should probably use the standard testthat format. see https://github.com/statnet/lolog/tree/master/tests as an example.